.circleci/config.yml

```diff
@@ -1272,8 +1270,10 @@ workflows:
               # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
               ignore: /pull\/[0-9]+/
       - backend-integration-tests-offline:
+          xcode_version: "15.2"
```

Comment:
The base job uses 15.3 and iOS 17.4 which is why the simulator can't be found in https://circleci.com/gh/RevenueCat/purchases-ios/231400

Reply:
THANK YOU!!! I was just trying to figure that out 😅